### PR TITLE
📝 Update documentation

### PR DESCRIPTION
**Pull request by asimpleidea**
This PR brings updates to the documentation:

- Fixes some minor typos
- `--kubeconfig` is now supported
- `NetworkPolicy` usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +51,8 @@ The project is now ready to be used locally from `./bin` directory as
 
 For a quick test of the project, you can try the `scripts/quickstart.sh` script
 included in this repository that will guide you through deploying *Egress
-Watcher* iteratively with some default values.
+Watcher* iteratively with some default values and working with a `ServiceEntry`
```
Comment (arnatal) [inline README.md:54]:
```suggestion
Watcher* iteratively with some default values and working with an Istio's `ServiceEntry`
```

File: README.md [hunk 2]
```diff
@@ -18,10 +18,11 @@ Feel free to reach out with any comment or question, you can find us at: cnwan@c
 ### Supported Egress types
 
 As of now, we support egress hosts defined as *ISTIO* `ServiceEntry` objects
+or as IPs defined in the `Egress` fields of a *Kubernetes* `NetworkPolicy`,
 and we reflect the changes we detect in them.
 
-Though only *ISTIO* is supported as of now, the project's architecture is
-designed to accomodate different types as defined by other projects.
+The project is designed to accomodate different types defined by other
```
Comment (arnatal) [inline README.md:24]:
```suggestion
The project is designed to accomodate different types of egress policies or external services defined by other
```

File: README.md [hunk 3]
```diff
@@ -137,11 +137,18 @@ Run the watcher:
 ```
 
 Try to deploy a `ServiceEntry` object you can use the provided example in
```
Comment (arnatal) [inline README.md:139]:
```suggestion
Try to deploy a `ServiceEntry` object. You can use the provided example in
```

File: README.md [hunk 4]
```diff
@@ -137,11 +137,18 @@ Run the watcher:
...
-`artifacts/yamls/istio`:
+`artifacts/yamls/examples/istio`:
 
 ```bash
 # In another shell terminal
-kubectl create -f ./artifacts/yamls/istio
+kubectl create -f ./artifacts/yamls/examples/istio
+```
+
+or a `NetworkPolicy`:
```
Comment (arnatal) [inline README.md:147]:
```suggestion
For a `NetworkPolicy` you can do instead:
```

**Review by arnatal — APPROVED** [review]
This looks pretty good to me @asimpleidea, thanks! One question, shall we update all the occurrences of 'ISTIO' to 'Istio'? It seems that the latter is the more common spelling?

**Comment by asimpleidea** [comment]
Merging. Thanks for the suggestions!

**Pull request merged**